org.onionshare.OnionShare.yaml

```diff
@@ -0,0 +1,336 @@
+          - make
+          - make install
+        sources:
+          - type: archive
+            url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+            sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+  - name: obfs4proxy
+    buildsystem: simple
+    build-options:
+      env:
+        GOBIN: "/app/bin/"
+        GO111MODULE: "off"
+    build-commands:
+      - ". /usr/lib/sdk/golang/enable.sh; GOPATH=$PWD go install gitlab.com/yawning/obfs4.git/obfs4proxy"
+    sources:
+      - type: git
+        url: https://go.googlesource.com/net
+        commit: 5f55cee0dc0dc168ce29222f077fe7fcd4be72c5
+        dest: src/golang.org/x/net
+      - type: git
+        url: https://go.googlesource.com/crypto
+        commit: 5ea612d1eb830b38bc4e914e37f55311eb58adce
+        dest: src/golang.org/x/crypto
+      - type: git
+        url: https://go.googlesource.com/sys
+        commit: 9a76102bfb4322425a1228caa377974426e82c84
+        dest: src/golang.org/x/sys
+      - type: git
+        url: https://go.googlesource.com/text
+        commit: 8f690f22cf1c026c950adddf3d45258bfd0912f0
+        dest: src/golang.org/x/text
+      - type: git
+        url: https://gitlab.com/yawning/utls
+        commit: 2dd4f38ff9e07464eb2748cc017eac1355e42251
+        dest: src/gitlab.com/yawning/utls.git
+      - type: git
+        url: https://gitlab.com/yawning/obfs4
+        commit: f638c33f6c6f697498150d5f0dfbf26453759262
+        dest: src/gitlab.com/yawning/obfs4.git
+      - type: git
+        url: https://gitlab.com/yawning/bsaes
+        commit: 0a714cd429ec754482b4001e918db30cd2094405
+        dest: src/gitlab.com/yawning/bsaes.git
+      - type: git
+        url: https://github.com/dchest/siphash
+        commit: a21c2e7914a8fe0db087fa007cbe804967665dfc
+        dest: src/github.com/dchest/siphash
+      - type: git
+        url: https://github.com/dsnet/compress
+        commit: da652975a8eea9fa0735aba8056747a751db0bd3
+        dest: src/github.com/dsnet/compress
+      - type: git
+        url: https://git.torproject.org/pluggable-transports/goptlib
+        commit: 781a46c66d2ddbc3509354ae7f1fccab74cb9927
+        dest: src/git.torproject.org/pluggable-transports/goptlib.git
+  - name: onionshare
+    buildsystem: simple
+    ensure-writable:
+      - easy-install.pth
+    build-commands:
+      - python3 setup.py install --prefix=${FLATPAK_DEST}
+      - install -D -m0644 org.onionshare.OnionShare.appdata.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml
+      - install -D -m0644 org.onionshare.OnionShare.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/org.onionshare.OnionShare.svg
+      - install -D -m0644 org.onionshare.OnionShare.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: dir
+        path: ../desktop/src
+    modules:
+      - name: python3-qrcode
```

Comment:
There's a generator for pip too, if it makes the work simpler for you: https://github.com/flatpak/flatpak-builder-tools/tree/master/pip